changed.d

```diff
@@ -547,7 +841,13 @@ Please supply a bugzilla version
 
         // print the entire changelog history
         if (revRange.length)
+        {
             bugzillaChanges.writeBugzillaChanges(w);
+        }
+        if (revRange.length)
+        {
+            githubChanges.writeBugzillaChanges(w);
```

Comment:
I think you should distinguish between Bugzilla and GitHub changes here with a `$(GITHUB ####)` macro, otherwise Github issue numbers will get Bugzilla links.

Reply:
I think I fixed this!?